src/test/functional/search-user-test.ts

```diff
@@ -1,15 +1,25 @@
-import { config as testConfig, testAccounts } from '../config';
+import { config as testConfig} from '../config';
 import * as Assert from 'assert';
+import {randomData} from './shared/random-data';
+import {createUserWithRoles} from './shared/apiHelpers';
 
 Feature('Search User');
 
+const dashboardUserEMAIL= testConfig.TEST_SUITE_PREFIX + randomData.getRandomEmailAddress();
+const citizenUserEmail= testConfig.TEST_SUITE_PREFIX + randomData.getRandomEmailAddress();
+BeforeSuite(async () => {
+  await createUserWithRoles(dashboardUserEMAIL, testConfig.PASSWORD, testConfig.USER_FIRSTNAME, []);
+  await createUserWithRoles(citizenUserEmail, testConfig.PASSWORD, testConfig.USER_FIRSTNAME, [testConfig.USER_ROLE_CITIZEN]);
+});
+
 const incorrectEmailAddresses = new DataTable(['incorrectEmailAddress']);
 incorrectEmailAddresses.add(['email']); // adding records to a table
 incorrectEmailAddresses.add(['email@']);
 incorrectEmailAddresses.add(['email@com']);
 
 Data(incorrectEmailAddresses).Scenario('I as an user should be able to see proper error message if search text is not in the right format', ({I,current}) => {
-  I.loginAsSystemOwner();
+  //I.loginAsSystemOwner();
```

Comment:
```suggestion
```
Probably just worth removing this comment now.

Reply:
Removed